README.md

```diff
@@ -80,4 +74,14 @@ ansible-vault decrypt '<path-to-file-to-decrypt>'
 
 ```bash
 ansible localhost -m debug -a var='<variable-name>' -e '@<path-to-file>' --ask-vault-pass
- ```
+```
+
+## Updating encrypted variables
+
+The `bootstrap.yml` playbook can be re-run to update variable values.
+
+1. Decrypt the `CREDENTIALS_CONFIG.yml` file as per [here](#decryption).
```

Comment:
Really nitpicking here but maybe instead of having the `here` link, we just add the exact command that needs to be run:
```
ansible-vault decrypt 'CREDENTIAL_CONFIG.yml'
```